crates/typos-cli/src/file.rs

```diff
@@ -664,6 +745,53 @@ fn fix_buffer(mut buffer: Vec<u8>, typos: impl Iterator<Item = typos::Typo<'stat
     buffer
 }
 
+fn fix_file_name<'a>(
+    path: &std::path::Path,
+    file_name: &'a str,
+    fixes: impl Iterator<Item = (typos::Typo<'a>, usize)>,
+) -> Result<std::path::PathBuf, std::io::Error> {
+    let file_name = file_name.to_owned().into_bytes();
+    let new_name = fix_buffer(file_name, fixes);
+    let new_name = String::from_utf8(new_name).expect("corrections are valid utf-8");
+    let new_path = path.with_file_name(new_name);
+    Ok(new_path)
+}
+
+fn select_fix(typo: &typos::Typo<'_>) -> Option<usize> {
+    let corrections = match &typo.corrections {
+        typos::Status::Corrections(c) => c,
+        _ => return None,
+    };
+
+    if corrections.len() == 1 {
+        let confirmation = Confirm::new()
+            .with_prompt("Do you want to apply the fix suggested above?")
+            .default(true)
+            .show_default(true)
+            .interact();
+        return confirmation.map(|_| 0).ok();
```

Comment:
imo use of `?` would make this easier to read

```suggestion
        let confirmation = Confirm::new()
            .with_prompt("Do you want to apply the fix suggested above?")
            .default(true)
            .show_default(true)
            .interact()
            .ok()?;
        0
```